looker/definitions/ads.toml

```diff
@@ -94,3 +136,44 @@ friendly_name = "PPA Measurements"
 description = "Aggregated conversion data from Privacy-Preserving Attribution (PPA)"
 columns_as_dimensions = true
 submission_date_column = "submission_date"
+
+
+[data_sources.newtab_desktop_ad_metrics_hourly]
+from_expression = """
+(
+  SELECT
+    submission_date,
+    spoc_id,
+    position,
+    advertiser,
+    campaign_name,
+    title,
+    flight_id,
+    creative_type,
+    site_name,
+    zone_name,
+    country,
+    rate_type,
+    pid,
+    ad_url,
+    image_url,
+    external_param,
+    impressions,
```

Comment:
None of these are referenced in the metric above, can they be removed?

Reply:
@jaredsnyder Do I need to explicitly state them if I want to include in the Looker view?

Reply:
Ah that I'm not sure of, not sure how this interacts with looker honestly but I'd guess you're right.